content/christine_belzie/create-a-url-using-slugs.md

```diff
@@ -16,10 +16,6 @@ CatalogContent:
 [URL]: https://www.codecademy.com/resources/docs/general/url
 [slugify.online]: https://slugify.online/
 
-[Step 1 of URL slug]: https://github.com/Codecademy/ugc/main/content/christine_belzie/step1-of-slug.png
-[Step 2 of URL slug]: https://github.com/Codecademy/ugc/main/content/christine_belzie/step2-of-slug.png
-[Step 3 of URL slug]: https://github.com/Codecademy/ugc/main/content/christine_belzie/step3-of-slug.png
```

Comment:
Can we keep this format and replace with: https://github.com/Codecademy/articles/blob/main/create-a-url-using-slugs/`image-name`.png?raw=true

Reply:
Hi @yangc95. I couldn't make the replacement.